### PR TITLE
Fixes #352 RelatedItem: Adjust file icons when importing file as rela…

### LINKED ISSUE #352: RelatedItem: Adjust file icons when importing file as related items
see https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/348#discussion_r168698527

### PR DESCRIPTION
**Pull request by msevestre**
…ted items

### CODE REVIEW

File: src/OSPSuite.Core/Domain/Mappers/FileExtensionToIconMapper.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+using OSPSuite.Core.Extensions;
+using OSPSuite.Utility;
+
+namespace OSPSuite.Core.Domain.Mappers
+{
+   public interface IFileExtensionToIconMapper : IMapper<string, ApplicationIcon>
+   {
+   }
+
+   public class FileExtensionToIconMapper : IFileExtensionToIconMapper
```
Comment (msevestre) [inline src/OSPSuite.Core/Domain/Mappers/FileExtensionToIconMapper.cs]: That needs to be in its own file with so many cases

**Comment by msevestre** [comment]
@Yuri05 Please review when you have time

File: src/OSPSuite.Core/Domain/Mappers/FileExtensionToIconMapper.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+         if (extension.IsOneOf(Constants.Filter.MATLAB_EXTENSION, Constants.Filter.FIG_EXTENSION, Constants.Filter.MAT_EXTENSION))
+            return ApplicationIcons.Matlab;
+
+         if (extension.IsOneOf(Constants.Filter.R_EXTENSION, Constants.Filter.RD_EXTENSION))
+            return ApplicationIcons.R;
+
+         if (extension.IsOneOf(Constants.Filter.PKML_EXTENSION, Constants.Filter.RD_EXTENSION))
+            return ApplicationIcons.PKML;
+
```
Comment (Yuri05) [inline src/OSPSuite.Core/Domain/Mappers/FileExtensionToIconMapper.cs]: RD_Extension==>PKML??

**Comment by msevestre** [comment]
Lol. Code is wrong indeed but this case will never happen anyways because
it's exited on previous line. I'll change it for next commit

**Pull request merged**